hikey.mk

```diff
@@ -420,9 +420,33 @@ nvme-cleaner:
 ################################################################################
 # Flash
 ################################################################################
+define flash_help
+	@read -r -p "1. Connect USB OTG cable, the micro USB cable (press any key)" dummy
+	@read -r -p "2. Connect HiKey to power up (press any key)" dummy
+endef
+
+.PHONY: recovery
+recovery:
+	@echo "Enter recovery mode to flash a new bootloader"
+	@echo "Jumper 1-2: Closed (Auto power up)"
+	@echo "       3-4: Closed (Boot Select)"
+	@echo "       5-6: Open   (GPIO3-1)"
```

Comment:
I totally agree that GPIO3-1 doesn't say much, but I just copy/pasted from the docs so that we don't deviate from their nomenclature. You will also find the GPIO3-1 in HiKey user's guide (i.e, not only the "getting started guide"), so I'd like to keep this name here. I could expland it by adding "fastboot" to it also, but .. you're also getting into fastboot when you're in recovery mode, hence it could be a bit confusing I think and confusion is the last thing we need with this already tricky (according to me) way of dealing with various flash modes, cables and various time outs.